Xamarin.Forms.Platform.UAP/CollectionView/StructuredItemsViewRenderer.cs

```diff
@@ -4,18 +4,16 @@
 
 namespace Xamarin.Forms.Platform.UWP
 {
-	public class StructuredItemsViewRenderer : ItemsViewRenderer
+	public class StructuredItemsViewRenderer<TItemsView> : ItemsViewRenderer<TItemsView>
+		where TItemsView : StructuredItemsView
 	{
-		StructuredItemsView _structuredItemsView;
 		View _currentHeader;
 		View _currentFooter;
 
-		protected override IItemsLayout Layout { get => _structuredItemsView.ItemsLayout; }
+		protected override IItemsLayout Layout { get => ItemsView?.ItemsLayout; }
```

Comment:
Since this can be null should all references to it account for that?

https://github.com/xamarin/Xamarin.Forms/blob/4.3.0/Xamarin.Forms.Platform.UAP/CollectionView/StructuredItemsViewRenderer.cs#L163

https://github.com/xamarin/Xamarin.Forms/blob/4.3.0/Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs#L177

Reply:
In both cases we're guaranteed that the ItemsView exists, and by extension the ItemsLayout. I suppose if someone deliberately set the ItemsView's ItemsLayout to null it could fail, but in that case it _should_ fail.